src/components/MultiSelect/MultiSelect.stories.tsx

```diff
@@ -37,6 +38,10 @@ export default {
             options: Object.keys(MultiSelectType),
             control: { type: 'select' },
         },
+        size: {
+            options: ['s', 'm'],
```

Comment:
See comment below on union type. It would be better to use the enum type like this: `Object.values(MultiSelectSize),`